src/libraries/System.Windows.Extensions/src/System.Windows.Extensions.csproj

```diff
@@ -56,14 +56,17 @@
     <Compile Include="System\Xaml\Permissions\XamlAccessLevel.cs" />
   </ItemGroup>
   <ItemGroup>
+    <!-- Need to reference the ref which has an OS agnostic configuration for netcoreapp3.0. -->
+    <ProjectReference Include="..\..\System.Drawing.Common\ref\System.Drawing.Common.csproj" />
```

Comment:
@ericstj @safern is it expected that the impl of S.D.Common doesn't have an OS agnostic configuration?

Reply:
Looks like it's being harvested: https://github.com/dotnet/corefx/pull/39275/commits/8e9adad67a084834c7888e643fdd1e720a70b8d8